libvirt/resource_cloud_init.go

```diff
@@ -34,6 +34,11 @@ func resourceCloudInit() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"meta_data": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
You should also add `Computed: true` here because `meta_data` is is populated without explicitly specifying it.